pep-0654.rst

```diff
@@ -702,59 +700,65 @@ merged with the unhandled ``TypeErrors``.
-   ...
-   *ValueError: ExceptionGroup('eg', [ValueError(1), ExceptionGroup('nested', [ValueError(6)])])
-   *OSError: ExceptionGroup('eg', [OSError(3), ExceptionGroup('nested', [OSError(4)])])
-   Traceback (most recent call last):
-     File "<stdin>", line 3, in <module>
-   ExceptionGroup
-      ------------------------------------------------------------
-      Traceback (most recent call last):
-       File "<stdin>", line 12, in <module>
-       File "<stdin>", line 3, in <module>
-      ExceptionGroup: eg
-        ------------------------------------------------------------
-        ValueError: 1
-        ------------------------------------------------------------
-        ExceptionGroup: nested
-          ------------------------------------------------------------
-          ValueError: 6
-      ------------------------------------------------------------
-      Traceback (most recent call last):
-       File "<stdin>", line 3, in <module>
-      ExceptionGroup: eg
-        ------------------------------------------------------------
-        TypeError: 2
-        ------------------------------------------------------------
-        OSError: 3
-        ------------------------------------------------------------
-        ExceptionGroup: nested
-          ------------------------------------------------------------
-          OSError: 4
-          ------------------------------------------------------------
-          TypeError: 5
-   >>>
+    >>> try:
+    ...     raise ExceptionGroup(
+    ...         "eg",
+    ...         [
+    ...             ValueError(1),
+    ...             TypeError(2),
+    ...             OSError(3),
+    ...             ExceptionGroup(
+    ...                 "nested",
+    ...                 [OSError(4), TypeError(5), ValueError(6)])
+    ...         ]
+    ...     )
+    ... except *ValueError as e:
+    ...     print(f'*ValueError: {e!r}')
+    ...     raise e
+    ... except *OSError as e:
+    ...     print(f'*OSError: {e!r}')
+    ...     raise
+    ...
+    *ValueError: ExceptionGroup('eg', [ValueError(1), ExceptionGroup('nested', [ValueError(6)])])
+    *OSError: ExceptionGroup('eg', [OSError(3), ExceptionGroup('nested', [OSError(4)])])
+
+      | ExceptionGroup
+      |  with 2 sub-exceptions:
+      +-+---------------- 1 ----------------
+        | Traceback (most recent call last):
+        |   File "<stdin>", line 15, in <module>
+        |   File "<stdin>", line 2, in <module>
+        | ExceptionGroup: eg
+        |  with 2 sub-exceptions:
+        +-+---------------- 1.1 ----------------
+          | ValueError: 1
+          +---------------- 1.2 ----------------
+          | ExceptionGroup: nested
+          |    with one sub-exception:
+          +-+---------------- 1.2.1 ----------------
+            | ValueError: 6
+          +------------------------------------
+        +------------------------------------
+        +---------------- 2 ----------------
```

Comment:
I think there should only be one line of dashes here, and its should be aligned with the following item (`----- 2 -----`).